lighthouse-cli/chrome-finder.js

```diff
@@ -34,15 +37,17 @@ module.exports = {
 
     execSync(
       `${LSREGISTER} -dump` +
-      ' | grep -i \'google chrome canary.app$\'' +
+      ' | grep -i \'google chrome[ canary]\{0,7\}.app$\'' +
```

Comment:
This is much better for sure, I've applied the group to the regex. Using the option --select-chrome finds both canary and standard chrome in my mac.